dashboard/lib/policies/child_account.rb

```diff
@@ -132,12 +120,29 @@ def self.personal_account?(user)
     end
   end
 
+  def self.state_policies
```

Comment:
If this is not a private method, would we want to add unit tests?

Reply:
good idea! https://github.com/code-dot-org/code-dot-org/pull/59131/commits/900608644fc424f3b1e1c683333b7b570cd29946